contracts/HubPool.sol

```diff
@@ -126,26 +126,26 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     event WhitelistRoute(uint256 destinationChainId, address originToken, address destinationToken);
 
     event InitiateRefundRequested(
-        uint64 requestExpirationTimestamp,
-        uint64 unclaimedPoolRebalanceLeafCount,
         uint256[] bundleEvaluationBlockNumbers,
+        uint32 requestExpirationTimestamp,
```

Comment:
As discussed IRL, these won't get packed no matter the order, but it's totally fine to move them for style!